src/views/EditPerson/SoldierScreen/index.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useMemo } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import _ from "lodash";
+import SearchBarPerson from "../SearchBarPerson";
+import SoldierForm from "./SoldierForm";
+import styles from "./soldierScreen.style";
+import Avatar from "../../../components/shared/Avatar/index";
+import {
+  fetchSoldierRequest,
+  getLoadings,
+  getErrors,
+  getPerson,
+} from "../../../features/apiComponents/editSoldierTab/editSoldierTabSlice";
+import { useHistory } from "react-router-dom";
+import PropTypes from "prop-types";
+import { Typography, InputLabel, Link } from "@material-ui/core";
+// import faker from "faker";
+
+/**
+ * tab of soldier
+ */
+export default function SoldierScreen({ personalNumber }) {
+  const errorMessage = "לא נמצא חייל העונה למ.א שהוזן";
+  const personalNumberTitle = "מספר אישי:";
+  const searchBarTitle = "הקלד מספר אישי להשלמת פרטי החייל:";
+  const updateLinkMessage = "עדכן";
+  // jss styles
+  const classes = styles();
+  const dispatch = useDispatch();
+  let history = useHistory();
+  // Props from redux
+  const { fetchInProgress, searchSoldierError, person } = useSelector(
+    (state) => {
+      const person = getPerson(state);
+      return {
+        ...getLoadings(state),
+        ...getErrors(state),
+        person,
+      };
+    }
+  );
+  // Allowing to search soldier
+  const [personEditable, setPersonEditable] = useState(true);
```

Comment:
Rename to `searchEnabled` or negate.
With the current naming - if `personEditable=true` the person fields are not editable (the search bar is displayed and accepts input)